content/Projects/prometheus-api-client-python.md

```diff
@@ -0,0 +1,15 @@
+# prometheus-api-client-python
+
+This is a python wrapper for the Prometheus API. It also includes some tools for processing metric data using Pandas Dataframes.
+
+### Installation
+
+To install the latest release:
+
+`pip install prometheus-api-client`
+
+### Links:
+ * [Source](https://github.com/aicoe/prometheus-api-client-python)
```

Comment:
It looks like CI is failing because it wants all of the list items to be out-dented a bit
```suggestion
* [Source](https://github.com/aicoe/prometheus-api-client-python)
```